tracing/src/lib.rs

```diff
@@ -636,7 +636,7 @@
 //! ### Emitting `log` Records
 //!
 //! This crate provides two feature flags, "log" and "log-always", which will
-//! cause [spans] and [events] to emit `log` records. When the "log" feature is
+//! cause [spans][span] and [events][event] to emit `log` records. When the "log" feature is
```

Comment:
Can we wrap this at the same column as the other lines in this comment?

Reply:
I've fixed this!